scripts/cl/one_for_zero.py

```diff
@@ -53,3 +75,23 @@ def calc_test_case_with_next_sqrt_price(liquidity: sp.Float, sqrt_price_current:
     print(F"fee_amount_per_share: {fee_amount_per_share}")
 
     return expected_token_in, token_out, fee_amount_per_share
+
+def calc_test_case_with_next_sqrt_price_in_given_out(liquidity: sp.Float, sqrt_price_current: sp.Float, sqrt_price_next: sp.Float, swap_fee: sp.Float) -> Tuple[sp.Float, sp.Float, sp.Float]:
+    """ Computes and prints one for zero test case parameters when next square root price is known.
+    Assumes swapping for token in given out.
+    
+    Returns expected token out, token in after fee, and fee amount per share. 
+    """
+    expected_token_out = get_expected_token_in(liquidity, sqrt_price_current, sqrt_price_next)
+    token_in = get_token_out(liquidity, sqrt_price_current, sqrt_price_next)
```

Comment:
same question here, maybe we just need to rename the func?

Reply:
If we rename, it would break the logic for swaps out given in.

Suggesting this approach instead:
https://github.com/osmosis-labs/osmosis/pull/4131#discussion_r1092796608